app/api/views/briefs.py

```diff
@@ -1382,3 +1420,67 @@ def award_brief_to_seller(brief_id):
         db_object=brief)
 
     return jsonify(work_order=work_order), 200
+
+
+@api.route('/brief/<int:brief_id>/ask-a-question', methods=['POST'])
+@login_required
+@role_required('supplier')
+def supplier_asks_a_question(brief_id):
+    """seller asks a question (role=supplier)
+    ---
+    tags:
+        - brief
+    definitions:
+        BriefAwarded:
+            type: object
+            properties:
+                awardedSupplier:
+                    type: string
+    responses:
+        200:
+            description: Brief awarded successfully.
+            schema:
+                $ref: '#/definitions/BriefAwarded'
+        400:
+            description: Bad request.
+        403:
+            description: Unauthorised to award brief to seller.
+        500:
+            description: Unexpected error.
+    """
+    brief = briefs.get(brief_id)
+    if not brief:
```

Comment:
We may also need to check: the brief is still open, the questions period is still open, the brief is not withdrawn, and this supplier is allowed to ask questions (i.e. for invite only briefs, that they were invited - basically the same logic check as whether they can respond or not).

Reply:
fixed